Xamarin.Forms.Platform.iOS/CollectionView/TemplatedCell.cs

```diff
@@ -69,5 +69,21 @@ void ClearSubviews()
 				ContentView.Subviews[n].RemoveFromSuperview();
 			}
 		}
+
+		public override bool Selected
+		{
+			get => base.Selected;
+			set
+			{
+				base.Selected = value;
+
+				var element = VisualElementRenderer?.Element;
+
+				if (element != null)
```

Comment:
Do we have a check (if it is possible to get to this state) that when the element is set, it goes to the correct state?

Reply:
I don't understand the question.

Reply:
If at some later stage, we go `cell.Element = something`, will it go to the correct state? It looks like we only change state when selection changes. So if we are selected and then get an element, will we still end up in the right state?

Reply:
Oh, the situation you're describing doesn't happen. All the element creation is managed by the controller; there's no place for a user to later set the element of a cell. There isn't even a cell.Element to set.